week1/index.js

```diff
@@ -1,38 +1,97 @@
-import HTTP from 'http'
-import Path from 'path'
-
-import sendIndexHTML from './responses/sendIndexHTML'
-import sendPage2HTML from './responses/sendPage2HTML'
-import sendStylesCSS from './responses/sendStylesCSS'
-import sendText from './responses/sendText'
-
-const server = HTTP.createServer((request, response) => {
-	console.log(request.method, request.url)
-
-	switch (request.url) {
-	case '/':
-		sendIndexHTML(response)
-		break
-	case '/page2':
-		sendPage2HTML(response)
-		break
-	case '/styles.css':
-		sendStylesCSS(response)
-		break
-	default:
-		const extension = Path.extname(request.url)
-		if (extension === '') {
-			response.statusCode = 302
-			response.setHeader('Location', '/')
-		} else {
-			response.statusCode = 404
-			sendText(response, "File not found")
-		}
+'use strict';
+
+const http = require("http");
+
+const server = http.createServer(() => {
+
+	// console.log("Creating a new server");
+});
+
+server.on('connection', () => {
+
+	// console.log("Connected");
+
+});
+
+server.on('clientError', (err, socket) => {
+
+	if (err)
+
+		socket.end('HTTP/1.1 400 Bad Request\r\n\r\n');
+});
+
+server.on('request', (req, res) => {
+
+	console.log("request", req.url);
+
+	let state = 10;
+
+	if (req.url === "") {
+
+
+		req.url = window.location = "http://username.wix.com/sitename#/!page-url/123456";
```

Comment:
I think you might be confused about what Node.js is and how it works - `window.location` is undefined in the Node.js runtime because it's a property published by the browser.